pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -531,36 +560,28 @@ func (policy *regoEnforcer) updateMetadata(results map[string]interface{}) error
 	return nil
 }
 
-func (policy *regoEnforcer) enforce(enforcementPoint string, input map[string]interface{}) error {
+func (policy *regoEnforcer) enforce(enforcementPoint string, input map[string]interface{}) (map[string]interface{}, error) {
 	results, err := policy.query(enforcementPoint, input)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	allowed, err := policy.allowed(enforcementPoint, results)
 	if err != nil {
-		return err
+		return results, err
```

Comment:
why is results being returned if theres a non-nil error?
is there a case where we parse the results for an error?

Reply:
This is an error, and has been fixed.